apps/meteor/app/api/server/v1/misc.ts

```diff
@@ -492,6 +492,18 @@ declare module '@rocket.chat/rest-typings' {
 	}
 }
 
+const parseErrorObject = (err: any) => {
+	if (err instanceof Error) {
```

Comment:
should we use `isMeteorError` instead?

https://github.com/RocketChat/Rocket.Chat/blob/de50690c839f4ad75baf9f7e7a29e45a1119fcbe/packages/core-services/src/MeteorError.ts#L22

Reply:
huh didn't know that existed. Let's try